src/libs/actions/Link.js

```diff
@@ -35,28 +36,36 @@ function showGrowlIfOffline() {
  * @param {String} url
  */
 function openOldDotLink(url) {
-    if (showGrowlIfOffline()) {
-        return;
-    }
-
+    /**
+     * @param {String?} shortLivedAuthToken
```

Comment:
Our [desing doc](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs) suggests that, Optional parameters should be enclosed by [] e.g. @param {String} [optionalText].

Reply:
Updated